src/components/home-page/home/ExplorerCourses/explorer-courses.scss

```diff
@@ -52,7 +52,7 @@
 }
 .scroll-courses-container {
   .scroll-wrapper {
-    display: grid;
+    display: grid !important;
```

Comment:
Is this `important` necessary? could you refactor the code and fix the problem and not to use it?

Reply:
Yup.